File: scratch/test_db.py
import os
from supabase import create_client

supabase_url = os.environ.get("NEXT_PUBLIC_SUPABASE_URL")
supabase_key = os.environ.get("SUPABASE_SERVICE_ROLE_KEY")
supabase = create_client(supabase_url, supabase_key)

res = supabase.table("inventory_items").select("*").limit(20).execute()
for item in res.data:
    print(f"{item['item_id']}: equipped={item.get('equipped')}")

File: scratch/test_db.js
require('dotenv').config({ path: '.env' });
const { createClient } = require('@supabase/supabase-js');

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
const supabaseKey = process.env.SUPABASE_SERVICE_ROLE_KEY;
const supabase = createClient(supabaseUrl, supabaseKey);

async function check() {
  const { data, error } = await supabase.from('inventory_items').select('*').limit(100);
  if (error) console.error(error);
  else {
    const userItems = data.filter(i => !i.item_id.startsWith('kingdom-tile'));
    for (const item of userItems) {
      console.log(`${item.name} (${item.item_id}): equipped=${item.equipped}, user_id=${item.user_id}`);
    }
  }
}
check();
